apps/test/unit/code-studio/components/progress/ProgressDotTest.js

```diff
@@ -421,23 +421,6 @@ describe('ProgressDot component tests', () => {
         const result = renderer.getRenderOutput();
         expect(result.props.children[0].props.className).to.equal('fa fa-lock');
       });
-
-      it('does not show a locked icon on header when locked', () => {
-        renderer.render(
-          <ProgressDot
-            level={{
-              icon: undefined,
-              kind: LevelKind.puzzle
-            }}
-            status={LevelStatus.locked}
-            courseOverviewPage={/* false implies this is header progress */false}
-            saveAnswersBeforeNavigation={false}
-          />
-        );
-
-        const result = renderer.getRenderOutput();
-        expect(result.props.children[0].props.className).to.equal('');
-      });
```

Comment:
Turns out in the header view, levels that are locked do not end up having a status of LevelStatus.locked, and thus don't get a lock icon. Thus, this was testing a code path that never happens.